docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Configuring docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blob Storage And Repository Configuration
```

Comment:
This is the only subsection in `Create New Docker Repository`.
Maybe have these 2 subsections instead:
```

#### Blob Storage

#### Repository
```

Reply:
changes applied